__tests__/app.test.js

```diff
@@ -376,6 +376,16 @@ describe("GET /api/articles/:article_id", () => {
           expect(articles).toBeSortedBy("author",{descending: true ,coerce: true,});
         });
     });
+    test("200: Responds with articles sorted by 'votes' in ascending order when order=asc is specified", () => {
+      return request(app)
+        .get("/api/articles?sort_by=votes&order=asc")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          
+          expect(articles).toBeSortedBy("votes", { ascending: true, coerce: true });
```

Comment:
This is not how the docs describe testing if something is in ascending order. This object accepts descending and this should be given a value of false if we want to assert this behaviour.